lib/formatters.ts

```diff
@@ -36,3 +36,15 @@ export const formatDuration = (ms: number) => {
  */
 export const capitalize = (str: string) =>
   str.charAt(0).toLocaleUpperCase() + str.slice(1).toLocaleLowerCase();
+
+/**
+ * Takes in a date given by Spotify and returns it in month/day/year.
+ * @param date The date string from Spotify
+ * @returns The date in month/day/year
+ */
+export const formatDate = (date: string) => {
```

Comment:
Instead of manually slicing it, you should do `new Date()` and then `toLocaleDateString()` here, it's more robust. There's other places that do that manually, can you convert them to use the lib function?